.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
```

Comment:
Just to make sure, this is going to check GitHub Action version everyday?

Reply:
Yes it will check for updates to `GitHub Actions` daily.  But it does not mean it will update everyday.